gateway/src/main/java/io/zeebe/gateway/impl/broker/RequestRetryHandler.java

```diff
@@ -27,6 +27,7 @@
  * all partitions are tried. The request is retried only for specific errors such as connection
  * errors or resource exhausted errors. The request is not retried for time outs.
  */
+@SuppressWarnings("java:S119")
```

Comment:
:sweat_smile:

Reply:
SonarCloud complains that the type names don't conform to the format. I don't think we ever decided on one, but I suppose we could just change the format on sonarcloud and not have to suppress those warnings

Reply:
What type names we are talking here? That interface should be an adjective?